Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -13660,6 +13704,49 @@ def pan_os_delete_tag_command(args: dict) -> CommandResults:
     )
 
 
+def pan_os_get_audit_comment_command(args: dict) -> CommandResults:
+    """
+    executes the command pan-os-get-audit-comment to get the audit comment for a given policy rule.
+
+    Args:
+        args (dict): The command arguments.
+
+    Returns:
+        CommandResults: The command results with raw response, outputs and readable outputs.
+    """
+    if DEVICE_GROUP and not PRE_POST:
+        raise DemistoException(f'The pre_post argument must be provided for panorama instance')
+
+    rule_name = args.get("rule_name") or ""
+    rule_type = args.get("rule_type") or ""
+    params = build_audit_comment_params(
+        name=rule_name,
+        pre_post='rulebase' if VSYS else f'{PRE_POST.lower()}-rulebase',
+        policy_type=RULE_TYPES_MAP[rule_type],
+        xml_type='show',
+    )
+
+    raw_response = http_request(URL, 'GET', params=params)
+    comment = (raw_response["response"]["result"] or {}).get("entry", {}).get("comment", "") or ""
+    outputs = {
+        "rule_name": rule_name,
+        "rule_type": rule_type,
+        "comment": comment
+    }
+
+    return CommandResults(
+        raw_response=raw_response,
+        outputs=outputs,
+        readable_output=tableToMarkdown(
+            f'Audit Comment for Rule: {rule_name}',
+            outputs,
+            headerTransform=string_to_table_header,
+        ),
+        outputs_prefix='Panorama.AuditComment',
+        outputs_key_field='rule_name'
```

Comment:
is a rule name unique identifier or need the rule type here as well?

Reply:
Probably need also by type. Added.